package.json

```diff
@@ -1,31 +1,30 @@
 {
   "name": "probot-linter",
   "version": "1.0.0",
-  "description": "{{{ description }}}",
-  "author": "{{{ author }}}",
+  "description": "automatically lints code",
+  "author": "Bex Warner and Brandon Keepers",
```

Comment:
I think author is intended to be one person (which is weird): https://docs.npmjs.com/files/package.json#people-fields-author-contributors

You've done more on this plugin than I have at this point, so feel free to just set your name there. 😁

Reply:
Super weird. What about using the contributors field instead?

```
"contributors": [
    {
      "name": "name"
    },
    {
      "name": "name"
    }
],
```
There are more fields that can be included in there, but they don't really matter.

Reply:
👍 works fo rme